### PR TITLE
Build with PortAudio support if the "portaudio" CONFIG flag is specified

### PR DESCRIPTION
**Pull request by davidebeatrici**

### CODE REVIEW

**Comment by mkrautz** [comment]
We don't recommend the use of the PortAudio backend in production. It's not as mature as the other backends. Though I understand that you're having issues with PulseAudio that aren't there in the PortAudio backend.

I'd be happier if this was a CONFIG flag. I don't have a particular thing in mind. Simply CONFIG+=portaudio wouldn't work. (Pulse's config block would still disable it.)

**Comment by mkrautz** [comment]
This possibly breaks OSX Universal. We'll need to redo the build script.

**Comment by davidebeatrici** [comment]
Because it relies on PortAudio being automatically enabled?

**Comment by mkrautz** [comment]
Yes. Could you do a PR against https://github.com/mumble-voip/mumble-releng/blob/master/buildscripts/1.3.x/mumble-osx-universal.bash to add it to the CONFIG lines?

**Comment by mkrautz** [comment]
Hmmm. Not actually sure. Maybe we use CoreAudio on that build now?

**Comment by davidebeatrici** [comment]
Sure.

File: src/mumble/mumble.pro [hunk 1]
```diff
@@ -446,12 +446,11 @@ unix {
   HAVE_PULSEAUDIO=$$system(pkg-config --modversion --silence-errors libpulse)
   HAVE_PORTAUDIO=$$system(pkg-config --modversion --silence-errors portaudio-2.0)
 
-  !isEmpty(HAVE_PORTAUDIO):!CONFIG(no-portaudio) {
+  !isEmpty(HAVE_PORTAUDIO):CONFIG(portaudio) {
```
Comment (mkrautz) [inline src/mumble/mumble.pro]: This doesn't make any sense. If CONFIG(config)... add it again? :-)

We can drop this whole section.

Drop HAVE_PORTAUDIO.
Drop this check.

Comment (davidebeatrici) [reply src/mumble/mumble.pro]: I read it as "Enable PortAudio" instead of "Add flag"...

**Comment by mkrautz** [comment]
@davidebeatrici Could you squash?

**Comment by mkrautz** [comment]
@davidebeatrici Also, will you do the mumble-releng PR?

**Pull request merged**